py/mpthread.h

```diff
@@ -58,4 +60,10 @@ void mp_thread_mutex_unlock(mp_thread_mutex_t *mutex);
 #define MP_THREAD_GIL_EXIT()
 #endif
 
+// for `-X realtime` command line option
+#if defined(__APPLE__)
+extern bool mp_thread_is_realtime_enabled;
+void mp_thread_set_realtime(void);
+#endif
```

Comment:
IMO this chunk should go in `ports/unix/mpthreadport.h`.

Reply:
done